packages/admin-panel/src/rootReducer.js

```diff
@@ -5,19 +5,19 @@
 import { combineReducers } from 'redux';
 import { reducer as authentication, LOGOUT } from './authentication';
 import { reducer as tables } from './table';
-import { reducer as autocomplete } from './autocomplete';
+import { reducer as autocomplete } from './autocomplete/reducer'; // Needs to be imported from reducer file or console shows autocomplete not found error
```

Comment:
This is slightly un-related but I noticed an error in the console and for some reason adding the import path fixes it.